readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
Actually, there's more that is problematic. You can't account for all ways to build up a valid value in a function, or composed results, etc. It's too restrictive and too unpredictable to predict all use cases. We can't even properly enforce the value case, right?